.github/workflows/deployment.yml

```diff
@@ -47,6 +47,13 @@ jobs:
         run: |
           echo '${{steps.fetch-contributors.outputs.result}}' > ${{ github.workspace }}/_data/contributors.json
           printf "contributors: %d\n" ${{steps.fetch-contributors.outputs.contributors-numof}} >> ${{ github.workspace }}/_data/riot_stats.yml
+      # Use GitHub Actions' cache to shorten build times and decrease load on servers
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
```

Comment:
All this change is part of an unrelated commit (the one about the base url)

Reply:
Right, should I split it into its own or merge it with 0fb5aa96e7773c8bcc20d86b19fb76cc7454fb7e ?

Reply:
I don't know. Is caching really needed for such a small site ? What is the performance gain ?

Reply:
When I first tried it it improved compared to not having it, I think it dropped from ~4 minutes to less than 2 minutes deploy time. It is also used in the example of the jekyll action (https://github.com/helaili/jekyll-action).

Reply:
This comment still needs to be addressed. I think this change should be in its own commit.

(And this PR needs squashing BTW)

Reply:
Will do

Reply:
Split in edbc5d6

Reply:
+1 for caching given the performance numbers.